src/kern/npf_tableset.c

```diff
@@ -456,13 +456,18 @@ npf_table_getid(npf_table_t *t)
  * npf_table_check: validate the name, ID and type.
  */
 int
-npf_table_check(npf_tableset_t *ts, const char *name, uint64_t tid, uint64_t type)
+npf_table_check(npf_tableset_t *ts, const char *name, uint64_t tid, uint64_t type, bool replacing)
 {
+	const npf_table_t *t;
```

Comment:
Please leave a trailing line between the variable declarations and the rest of the function body:
```c
	const npf_table_t *t;

	...
```